ffi/capi/dart/package/test/test.dart

```diff
@@ -0,0 +1,25 @@
+import 'package:icu/lib.dart';
+import 'package:path/path.dart' as path;
+import 'package:test/test.dart';
+
+void main() {
+  init(path.absolute("test/libicu_capi_cdylib.so"));
```

Comment:
Don't mix quote types, maybe think about enabling `prefer_single_quotes` in the analysis options.

Reply:
(You might want to enable a collection of lints, such as https://pub.dev/packages/dart_flutter_team_lints